lang/step.md

```diff
@@ -69,7 +69,7 @@ impl<M: Memory> Machine<M> {
     }
 
     fn eval_value(&mut self, ValueExpr::Constant(constant, _ty): ValueExpr) -> NdResult<Value<M>> {
-        self.eval_constant(constant)
```

Comment:
This code *seemed* correct, but when assembled into a match block it caused an error.
The other match arms return `Value`, which is expected to be converted to `NdResult` by `specr::hidden::ret`.